docs/packages/clean/README.md

```diff
@@ -70,6 +70,23 @@ module.exports = {
 };
 ```
 
+## Usage to clean directories outside of the project root
+
+By setting the value of allowExternal to true, clean middleware can be used to remove directories outside of the project directory. This option is set to false by default to avoid unintended deletion of directories.
+
+```js
+
+module.exports = {
+  options: {
+    // override the output directory
+    output: '../build'
+  },
+  // set allowExternal to true 
+  allowExternal: true,
+  use: ['neutrino-preset-react']
```

Comment:
Could you update these `use` references to our current middleware, `@neutrinojs/react`?